doc/index.rst

```diff
@@ -74,10 +74,30 @@ and simply replace ``'default.qubit'`` with any of the available Qiskit devices,
 You can also try to run tutorials, such as the qubit rotation tutorial, on actual quantum hardware by
 using the ``'qiskit.ibmq'`` device.
 
-To filter tutorials that explicitly use a qiskit device, use the "Qiskit" filter on the right panel of the
-`demos <https://pennylane.ai/qml/demonstrations.html>`_.
+A few of the PennyLane demos make extensive use of the plugin:
+
+.. raw:: html
+
+    <div class="row">
+
+.. demogalleryitem::
+    :name: Ensemble classification with Forest and Qiskit devices
+    :figure: _static/ensemble_diagram.png
```

Comment:
Out of curiosity, does it work if we include the URL to the figure? E.g., https://pennylane.ai/qml/_images/ensemble_diagram1.png?

This avoids needing to copy all thumbnails into the plugin docs, but I'm not sure if it will work---the `demogalleryitem` directive might need to be updated for this.

Reply:
Great idea. I adjusted the directive and it works no problem. We can't do it yet for the QV one though since it's not in the gallery.